test/controllers/v1/relay_group_membership_controller_test.exs

```diff
@@ -48,6 +46,20 @@ defmodule Cog.V1.RelayGroupMembershipControllerTest do
     assert member["name"] == relay.name
   end
 
+  test "relay group includes assigned bundles", %{authed: requestor} do
+    bundle = bundle("foo")
+    relay_group = relay_group("test-relay-group-1")
+    assign_bundle_to_group(relay_group.id, bundle.id)
+    conn = api_request(requestor, :get, "/v1/relay_groups/#{relay_group.id}")
+    fetched = json_response(conn, 200)["relay_group"]
+    assert relay_group.id == fetched["id"]
+    assert relay_group.name == fetched["name"]
+    assert length(fetched["bundles"]) == 1
+    member = Enum.at(fetched["bundles"], 0)
+    assert member["id"] == bundle.id
+    assert member["name"] == bundle.name
```

Comment:
@imbriaco if you're using a strict equality operator (`==`) that's true. If what I wrote out is the entirety of the result that comes back, I'd use that, which means we could use things like `bundle.id` directly. However, if there are other keys in either of those maps that are not currently being matched on, `==` won't work.